documentation/config_logging_advanced.md

```diff
@@ -4,123 +4,131 @@ title: "Puppet Server: Advanced Logging Configuration"
 
 #### Adding a JSON version of the main Puppet Server logs
 
-To configure Puppet Server to log its main logs to a second log file, in JSON format, simply add a section like this to your `logback.xml` file (it should be added at the same level in the XML as your existing appender/appenders, but the order of the appenders does not matter):
+Logback writes logs using components called [appenders](http://logback.qos.ch/manual/appenders.html). The example code below uses `RollingFileAppender` to rotate the log files and avoid consuming all of your storage.
 
-``` xml
-<appender name="JSON" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>/var/log/puppetlabs/puppetserver/puppetserver.log.json</file>
+1. To configure Puppet Server to log its main logs to a second log file in JSON format, add an appender section like the following example to your `logback.xml` file, at the same level in the XML as existing appenders. The order of the appenders does not matter.
 
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>/var/log/puppetlabs/puppetserver/puppetserver.log.json.%d{yyyy-MM-dd}</fileNamePattern>
-            <maxHistory>5</maxHistory>
-        </rollingPolicy>
+    ``` xml
+    <appender name="JSON" class="ch.qos.logback.core.rolling.RollingFileAppender">
+            <file>/var/log/puppetlabs/puppetserver/puppetserver.log.json</file>
 
-        <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
-    </appender>
-```
+            <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+                <fileNamePattern>/var/log/puppetlabs/puppetserver/puppetserver.log.json.%d{yyyy-MM-dd}</fileNamePattern>
+                <maxHistory>5</maxHistory>
+            </rollingPolicy>
 
-Logback writes logs using components called [appenders](http://logback.qos.ch/manual/appenders.html).  Note that the above configuration includes the `RollingFileAppender`, to rotate the log files and ensure they don't fill up your disk.  Also, note that there are a lot of configuration options available for the `LogstashEncoder` above, including the ability to modify the list of fields that you wish to include, or give them different field names.  For more info see the [Logstash Logback Encoder Docs](https://github.com/logstash/logstash-logback-encoder/blob/master/README.md#loggingevent-fields).
+            <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+        </appender>
```

Comment:
When the markdown is rendered here, the `<appender>` `</appender>` tags see to not be aligned. Don't know if we care.